common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -274,7 +274,9 @@ def import_static(self, data_path, dest_id):
         Import all static items into the content store.
         """
         if self.static_content_store is None:
-            log.warning("Static content store is None. Skipping static content import...")
+            log.warning(
+                "Course import %s: Static content store is None. Skipping static content import...", self.target_id
```

Comment:
log.error(`CourseImportError: {courselike_key}:....`

Reply:
I thought warning was better because it does not fail the course import

Reply:
I don't think Splunk captures `warning` logs?

Reply:
I though it did. I changed it anyway